src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4983,7 +5002,26 @@ public override bool PreVisit(CallNode node)
                 _currentScope = scopeNew.Parent;
                 PostVisit(node.Args);
 
-                // Typecheck the invocation.
+                if (maybeFunc.HasIdentifiers && _features.HasFlag(Features.SupportIdentifiers))
+                {
+                    var i = 0;
+
+                    foreach (var arg in args)
+                    {
+                        i++;
+
+                        if (arg is FirstNameNode firstNameNode && maybeFunc.IsIdentifierParam(i))
+                        {
+                            if (DType.TryGetLogicalNameForColumn(argTypes[0], firstNameNode.Ident.Name, out var logicalName))
+                            {
+                                var typedName = argTypes[0].GetAllNames(DPath.Root).First(tn => tn.Name == logicalName);
+                                var fileNameInfo = FirstNameInfo.Create(firstNameNode, new NameLookupInfo(BindKind.NamedValue, typedName.Type, DPath.Root, 0, displayName: new DName(logicalName)));
```

Comment:
>logicalName [](http://example.com/codeflow?start=181&length=11)

Why are we passing something called "logicalName" to a field for DisplayName?

Reply:
Please allow me this incongruity...
The code is full of that already, one more, one less, will it make any difference?
I've fixed it though, it's better to use a valid name (and all the code should be updated too)

Taking this one as it's in front of me, why is something that "visits" called "accept" ?!

        /// <summary>
        /// This method visits the node using the visitor and context provided.
        /// </summary>
        /// <typeparam name="TResult"></typeparam>
        /// <typeparam name="TContext"></typeparam>
        /// <param name="visitor">Visitor to use.</param>
        /// <param name="context">Context.</param>
        /// <returns></returns>
        public abstract TResult Accept<TResult, TContext>(IRNodeVisitor<TResult, TContext> visitor, TContext context);